lib/ace/ext/modelist.js

```diff
@@ -122,6 +122,7 @@ var supportedModes = {
     Pascal:      ["pas|p"],
     Perl:        ["pl|pm"],
     pgSQL:       ["pgsql"],
+    PL/SQL:      ["PL/SQL"],
```

Comment:
this should have name of the mode and common extensions used for that mode. something like `plSQL: ["pls"] `